src/assets/icons/ArrowBottomIcon.tsx

```diff
@@ -0,0 +1,8 @@
+export const ArrowBottomIcon = () => {
+    return (
+        <svg width="14" height="9" viewBox="0 0 14 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M1.06661 0.80994L6.99963 6.87618L13.0659 0.943152" stroke="black" stroke-opacity="0.3" stroke-width="2"/>
```

Comment:
Rename attributes in all icons on camelcase (stroke-opacity must be strokeOpacity etc)